lib/index.js

```diff
@@ -1,8 +1,10 @@
-import isEmpty from 'lodash/isEmpty';
-import platform from 'platform';
+import {detect} from 'detect-browser';
+import {FetchServer} from './fetch-server';
 
-export {FetchServer} from './fetch-server';
+export {FetchServer};
 
+// https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_isempty
+const isEmpty = (obj) => [Object, Array].includes((obj || {}).constructor) && !Object.entries(obj || {}).length;
```

Comment:
🤷‍♀️ if having lodash as a dep was making something hard about this change then ok, but otherwise let's not maintain our own implementation of `isEmpty`

Reply:
the lodash does add some complexity.. `lodash` module is published as commonjs and `lodash-es` as esm module. So in esm mode `import isEmpty from 'lodash/isEmpty'` will throw an error cause `lodash/isEmpty.js` is a commonjs module.

Reply:
couldn't we use `lodash-es` tho?

Reply:
that's the complication:

if use `lodash-es` then the commonjs export `.cjs` will not work properly 🙈

Reply:
updated to use `lodash-unified`~